src/exodus/walletmodels.h

```diff
@@ -83,31 +77,42 @@ class SigmaMintId
 class SigmaMint
 {
 public:
-    PropertyId property;
-    DenominationId denomination;
+    SigmaMintId id;
+
+    int32_t count;
```

Comment:
Seem like it should be never negative. So we should change it type to `uint32_t` instead.